pyvo/mivot/utils/exceptions.py

```diff
@@ -25,6 +25,16 @@ class MappingException(Exception):
     """
 
 
+class NoMatchingDMType(Exception):
```

Comment:
```suggestion
class NoMatchingDMTypeError(Exception):
```

Reply:
I would also think it should be a subclass of either TypeError or ValueError, so this is not a strong preference.